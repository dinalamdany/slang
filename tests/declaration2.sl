/* Test double declaration of a variable 
local scoping
*/

main(){
	int a = 100;
    print(a);
	a = 5;
    print(a);
	init{
		int b;
		b = 200;
        print(b);
		int a = 10;
        print(a); /* variable already delcared, what should happen? */
	}
}
