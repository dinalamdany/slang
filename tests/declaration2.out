100
5
200
10
